core/io/dir_access.cpp

```diff
@@ -222,7 +222,7 @@ String DirAccess::fix_path(String p_path) const {
 	return p_path;
 }
 
-DirAccess::CreateFunc DirAccess::create_func[ACCESS_MAX] = { nullptr, nullptr, nullptr };
+DirAccess::CreateFunc DirAccess::create_func[ACCESS_MAX] = { nullptr, nullptr };
```

Comment:
I'm not sure about this change, why two entries for an array with three entries? I'd say it's wrong in `FileAccess`, not here

Reply:
> I'm not sure about this change, why two entries for an array with three entries? I'd say it's wrong in `FileAccess`, not here

So I should change FileAccess instead of here?

Reply:
I'd say yes, not sure this causes any problems, but it's probably not good either to have this 2/3 thing

Reply:
> I'd say yes, not sure this causes any problems, but it's probably not good either to have this 2/3 thing

Done